cli/prog/sigstore.py

```diff
@@ -115,12 +115,13 @@ def create_sigstore(data):
 @create_sigstore.group("root_of_trust", invoke_without_command=True)
 @click.option("--root_name", required=True, help="Root of trust name")
 @click.option("--private", is_flag=True, default=False, help="Set this flag for private root of trust")
+@click.option("--rootless_keypairs_only", is_flag=True, default=False, help="Set this flag to only allow explicit keypair verifiers")
```

Comment:
Please make it clear that "rootless_keypairs_only" setting overrides "is_private" setting in the help.